src/characters/alice.ts

```diff
@@ -185,4 +187,51 @@ export class Alice {
       policyEndTime
     ) as BlockchainPolicyParameters;
   }
+
+  public async revoke(
+    policy: EnactedPolicy
```

Comment:
Considering that only `policyID` is used from `EnactedPolicy`, and alice may be revoking this policy without having any good way to instantiate an EnactedPolicy in a separate runtime form the granting operation, perhaps we just accept the policy ID and not an entire Policy here.

Reply:
But don't we need other things such as revocation kits for off-chain revocation? This is not exactly a priority in this issue

Reply:
How about:
```
 public async revoke(
    // provide atleast one:
    policyId?: Uint8Array, // to revoke on-chain 
    revocationKit?: RevocationKit // to revoke off-chain
  ) 
```